deploy/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/openshift/origin-operator-registry:latest
```

Comment:
I don't think that deploy/Dockerfile is the right home for this. IMO this location communicates that it is required for deployments, which this is not

Reply:
Fair enough, this is also something we saw in other projects, please advice where to place it

Reply:
hmm this will likely have to go in the root and be renamed to `Dockerfile.olm` or something of the sort

Reply:
done